src/services/AccommodationService.js

```diff
@@ -0,0 +1,40 @@
+import { accommodations, rooms, accommodationImages } from '../database/models';
+import CommonQueries from './CommonQueries';
+
+/**
+ * @exports
+ * @class AccommodationService
+ */
+class AccommodationService {
+  /**
+   * Travel Admin can be able to create accomodation
+   * @static
+   * @param {object} req  request object
+   * @memberof AccommodationService
+   * @returns {object} data
+   */
+  static async createAccomodation(req) {
+    const newAccommodationObject = {
+      name: req.body.name,
+      cityId: req.body.cityId,
+      address: req.body.address,
+      description: req.body.description,
+      googleCoordinates: req.body.googleCoordinates,
+    };
+    const newAccommodation = await CommonQueries.create(accommodations, newAccommodationObject);
+    Array.from(new Set(req.body.imageUrls)).map(async (imageUrl) => {
```

Comment:
I think you should not have used ~~async~~ (imageUrl) because the function itself behave asynchronously.